### PR TITLE
Document underscored: true effect on table name inference

### PR DESCRIPTION
**Pull request by barraponto**
Setting `{underscored: true}` on a model will also effect the table name inference.
A model called `UserProfile` will have the table name set to `user_profiles` unless explicitly overriden.
Let's make this clearer in the documentation.

### CODE REVIEW

File: docs/other-topics/naming-strategies.md [hunk 1]
```diff
@@ -21,12 +21,14 @@ Above we have the models User and Task, both using the `underscored` option. We
 
 Without the `underscored` option, Sequelize would automatically define:
 
+* An `Users` table for the User model and a `Tasks` table for the Task model.
```
Comment (ephys) [inline docs/other-topics/naming-strategies.md:24]:
```suggestion
* A `Users` table for the User model and a `Tasks` table for the Task model.
```

File: docs/other-topics/naming-strategies.md [hunk 2]
```diff
@@ -21,12 +21,14 @@ Above we have the models User and Task, both using the `underscored` option. We
...
 Without the `underscored` option, Sequelize would automatically define:
 
+* An `Users` table for the User model and a `Tasks` table for the Task model.
 * A `createdAt` attribute for each model, pointing to a column named `createdAt` in each table
 * An `updatedAt` attribute for each model, pointing to a column named `updatedAt` in each table
 * A `userId` attribute in the `Task` model, pointing to a column named `userId` in the task table
 
 With the `underscored` option enabled, Sequelize will instead define:
 
+* An `users` table for the User model and a `tasks` table for the Task model.
```
Comment (ephys) [inline docs/other-topics/naming-strategies.md:31]:
```suggestion
* A `users` table for the User model and a `tasks` table for the Task model.
```

File: docs/other-topics/naming-strategies.md [hunk 3]
```diff
@@ -4,13 +4,13 @@ title: Naming Strategies
 
 ## The `underscored` option
 
-Sequelize provides the `underscored` option for a model. When `true`, this option will set the `field` option on all attributes to the [snake_case](https://en.wikipedia.org/wiki/Snake_case) version of its name. This also applies to foreign keys automatically generated by associations and other automatically generated fields. Example:
+Sequelize provides the `underscored` option for a model. When `true`, this option will set the `field` option on all attributes to the [snake_case](https://en.wikipedia.org/wiki/Snake_case) version of its name. This also applies to table names and foreign keys automatically generated by associations and other automatically generated fields. Example:
```
Comment (ephys) [inline docs/other-topics/naming-strategies.md:7]: I think it's important to mention that this only applies if the `tableName` has not been provided. If it was provided, it's used as-is

**Review by ephys — CHANGES_REQUESTED** [review]
Thank you for taking the time to improve the documentation!

File: docs/other-topics/naming-strategies.md (hunk 3 above)
Comment (barraponto) [reply docs/other-topics/naming-strategies.md:7]:
```suggestion
Sequelize provides the `underscored` option for a model. When `true`, this option will set the `field` option on all attributes to the [snake_case](https://en.wikipedia.org/wiki/Snake_case) version of its name (unless manually set). This also applies to table names and foreign keys automatically generated by associations and other automatically generated fields. Example:

```

Comment (ephys) [reply docs/other-topics/naming-strategies.md:7]: Yes that sounds good to me :)

Comment (barraponto) [reply docs/other-topics/naming-strategies.md:7]: yeah, cause it actually applies to fields as well, right? you can manually set those too.

**Comment by barraponto** [comment]
@ephys is there any thing else missing? I'm eager to see this merged :)

File: docs/other-topics/naming-strategies.md (hunk 3 above)
Comment (WikiRik) [reply docs/other-topics/naming-strategies.md:7]: @barraponto can you still commit this change?

Comment (barraponto) [reply docs/other-topics/naming-strategies.md]: I was under the impression it was commited already 🤯 

done.

**Review by WikiRik — APPROVED** [review]

**Pull request merged**